api/src/test/java/io/minio/MinioClientTest.java

```diff
@@ -533,7 +534,7 @@ public void testPutSmallObjectFails()
 
   @Test(expected = EOFException.class)
   public void testPutIncompleteSmallPut()
-    throws NoSuchAlgorithmException, InvalidKeyException, IOException, XmlPullParserException, MinioException {
+      throws NoSuchAlgorithmException, InvalidKeyException, IOException, XmlPullParserException, MinioException {
```

Comment:
The tabbing is correct, previously it was wrong. Now it is consistent with rest of the code.